app/services/manual/preview_service.rb

```diff
@@ -6,6 +6,11 @@ def initialize(user:, manual_id:, attributes:)
   end
 
   def call
+    manual = if manual_id
+               Manual.find(manual_id, user)
+             else
+               Manual.new(attributes)
+             end
```

Comment:
Minor: I find these hanging indents hard to read. Is there a reason this can't remain a ternary expression?